variables.tf

```diff
@@ -105,6 +105,16 @@ variable "schemas" {
   default     = []
 }
 
+variable "schema_global_rule" {
+  type        = string
+  description = "Schema global compatibility rule. Only allowed for enterprise plan, must be '' for other plans. Allowed values are 'NONE', 'FULL', 'FULL_TRANSITIVE', 'FORWARD', 'FORWARD_TRANSITIVE', 'BACKWARD', 'BACKWARD_TRANSITIVE'."
```

Comment:
I guess the description needs to be updated actually. It says: `must be '' for other plans` - but I its actually `null`. I think maybe just remove it and say `Only allowed for enterprise plan`

Reply:
I have removed the line as a whole as we are doing plan validation for this already.